frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -1,17 +1,37 @@
+import { useCallback, useState } from '#libs/hooks/hooks.js';
+
 import styles from './styles.module.scss';
 
 type Properties = {
   label: string;
+  groupName?: string;
   onChange: () => void;
+  type?: 'checkbox' | 'radio';
+  isChecked?: boolean;
 };
 
-const Checkbox: React.FC<Properties> = ({ label, onChange }) => {
+const Checkbox: React.FC<Properties> = ({
+  groupName = 'checkbox',
+  label,
+  onChange,
+  type = 'checkbox',
+  isChecked,
+}) => {
+  const [checked, setChecked] = useState(isChecked);
```

Comment:
we should control it using useForm, the same flow as for Input component
```suggestion
```

Reply:
done